### PR TITLE
MEN-6671: Fix and white-list `test_fault_tolerance` tests

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline --pr mender/1486

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline --pr mender/1487

**Review by kacf — APPROVED** [review]

**Review by oleorhagen — APPROVED** [review]

File: tests/tests/test_fault_tolerance.py [hunk 1]
```diff
@@ -110,8 +107,7 @@ def wait_for_download_retry_attempts(self, device, search_string):
 
         while int(time.time()) < timeout_time:
             output = device.run(
-                "journalctl -u %s -l --no-pager | egrep 'msg=\".*%s' | wc -l"
-                % (device.get_client_service_name(), search_string),
+                f"journalctl -u mender-updated -l --no-pager | egrep 'msg=\".*{search_string}' | wc -l",
```
Comment (oleorhagen) [inline tests/tests/test_fault_tolerance.py:110]: Nit: `s/egrep/grep -E`

**Review by oleorhagen — APPROVED** [review]
Nice!

File: tests/tests/test_fault_tolerance.py (hunk 1 above)
Comment (lluiscampos) [reply tests/tests/test_fault_tolerance.py:110]: I have a pending PR with `grep -E`, long flags for journalctl, plus minor beautifications. Let me address everything there once the tests are _testing_

**Pull request merged**